website/events/templates/events/email/registration_conf_email.html

```diff
@@ -0,0 +1,10 @@
+{% extends "email/html_email.html" %}
+{% block content %}
+  <p>Dear {{ registration.member.profile.display_name }},</p>
+
+  <p>Hereby a confirmation that you registered for the following even: '{{ event.title }}', on:  {{ registration.date }}.</p>
+  <p>Please inform the website on any other possible information needed from you to complete your registration.</p>
```

Comment:
This line isn't needed when the event has no registration fields (that should be easy to check)